src/controllers/main/main.ts

```diff
@@ -1798,8 +1801,10 @@ export class MainController extends EventEmitter {
       },
       actionId
     )
-
-    this.onSignSuccess('account-op')
+    await this.#notificationManager.create({
+      title: 'Successfully signed',
+      message: 'Your transaction was successfully signed and broadcasted to the network'
+    })
```

Comment:
We can take the change to update the text (since the title and message repeat themselves). How about:

- title: Done!
- message: Transaction was signed and broadcasted to the network.